packages/ui-kit/src/components/Logger/LogProvider.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { createContext } from 'react'
+
+import { LogLevels } from './Log.types'
+
+export type LogContextProps = {
+  logLevel?: LogLevels
```

Comment:
Does it really need to be optional? Can't the object itself just be optional?
```suggestion
  logLevel: LogLevel
```